Application/Sources/UI/Views/LiveRadioSquaredCell.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  LiveRadioSquaredCell.swift
```

Comment:
```suggestion
//  LiveRadioCell.swift
```

So maybe follow first comment, it's a better name, contrary to `LiveMediaCell` which is for both video and audio livestreams.